scss/6-components/_skip-links.scss

```diff
@@ -0,0 +1,16 @@
+.cads-skip-links {
```

Comment:
Mind adding `/** define skip-links */` to this so it triggers the BEM lint rule? Although this looks good so don't expect any warnings.

Reply:
Sorry! I just realised I typoed this 🙈 , needs an `@` in `@define`